toolkit/mozapps/handling/ContentDispatchChooser.jsm

```diff
@@ -323,7 +323,7 @@ const replaySchemeMap = {
       toggleRecording(browser.ownerDocument.defaultView.gBrowser.selectedBrowser);
     }).catch((e) => {
       pingTelemetry("replay:record", "error", { url: target, newtab, message: e.message });
-      tabbrowser.loadURI(`https://app.replay.io/browser/error?message=Failed to launch recorder: (${e.message})`, {
+      tabbrowser.loadURI(`https://app.replay.io/browser/error?message=Failed to launch recorder: (${e.message})&url=${encodeURIComponent(url.asciiSpec)}`, {
```

Comment:
Should we just use `.spec` since we're encoding the value either way?

Also seems like we should be encoding `e.message` for this to be safe, otherwise a weird message could break parsing of the `url` param.